Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="118" judge-id="34" status="SE" scoring="ACM" run-tests="1" contest-id="3205" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="1" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>57da4a37-c461-424c-b4a5-fe1468b079df</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="SE" time="287" real-time="308" max-memory-used="18698240" >
  <program-stats-str>{ utime=246, stime=41, ptime=287, rtime=308, maxvsz=18698240, maxrss=19247104, nvcsw=1, nivcsw=40 }</program-stats-str>
      <input size="14">1 2 2 3 3 3 4
</input>
      <output size="6">2
2
4
</output>
      <stderr size="279">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000118&quot;, line 8, in &lt;module&gt;
    if a[n]%2==0:
IndexError: list index out of range
</stderr>
    </test>
  </tests>
</testing-report>
